rllib/evaluation/worker_set.py

```diff
@@ -71,6 +72,12 @@ def remote_workers(self):
         """Return a list of remote rollout workers."""
         return self._remote_workers
 
+    def sync_weights(self):
```

Comment:
Can we add a tau-option here for soft synching?

Reply:
This is just a convenience method for existing hard sync. Soft sync should be done at the local level not between workers.